app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,28 @@
+module SearchBarHelper
+  def mount_search_bar(
+    id,
+    controller: auto_complete_controller_name,
+    url: "#{auto_complete_controller_name}/auto_complete_search",
```

Comment:
FYI this broke the auto_complete feature for nested controllers (that we have in some plugins). The PR to fix it is here https://github.com/theforeman/foreman/pull/6460